src/storage/src/source/reclock/compat.rs

```diff
@@ -124,17 +133,31 @@ impl PersistHandle {
     ) -> anyhow::Result<Self> {
         let mut persist_clients = persist_clients.lock().await;
         let persist_client = persist_clients
-            .open(metadata.persist_location)
+            .open(metadata.persist_location.clone())
             .await
             .context("error creating persist client")?;
         drop(persist_clients);
 
+        let since_handle: SinceHandle<_, _, _, _, PersistEpoch> = persist_client
+            .open_critical_since(
+                metadata.remap_shard.clone(),
+                PersistClient::CONTROLLER_CRITICAL_SINCE,
+                // TODO(aljoscha): Use a different purpose? We should be fine,
```

Comment:
Probably just remove the TODO because the purpose is fine?

Reply:
IMO anything that is unique is fine